content/en/registry/sdk-swift-nautilustelemetry.md

```diff
@@ -0,0 +1,19 @@
+---
+title: NautilusTelemetry
+registryType: SDK
```

Comment:
AFAIK, we don't have an `sdk` registry type yet:

```console
$ find content/en/registry/*.* -exec grep -e 'registryType:' {} \; | sort | uniq
registryType: core
registryType: exporter
registryType: extension
registryType: instrumentation
registryType: processor
registryType: receiver
registryType: utilities
```

Is this a type that we want to add @austinlparker?

If so, it'll need to be in lowercase @ladd.